packages/react-form-state/src/components/tests/Nested.test.tsx

```diff
@@ -30,7 +30,7 @@ describe('<Nested />', () => {
 
     expect(renderPropSpy).toBeCalledWith({
       title: {
-        name: 'nodejs.title',
+        name: 'product.title',
```

Comment:
Just want to double check that it makes sense to update the test. I didn't understand where `nodejs` was coming from.

Reply:
yeah this makes sense 👍 that was coming form the bug you found probably